### DISCUSSION TITLE
Download Compliance Assessment

### DISCUSSION THREAD

**Discussion by capecemi** [CrowdStrike/falconpy#1185] [Q&A] [answered]
In the CrowdStrike console, under the Cloud Security/Cloud Compliance menu, I can view our Assessment, filtered to CIS Framework Version 1.4.0. Using falconpy, which Service Collection/Operation ID would I use to download this report?

**Comment by jshcodes** [comment, marked as answer]
Hi @capecemi -

The API operations for pulling this detail have just been released and will be available in the next release of FalconPy. 😁 

In the interim, you could also generate a Scheduled Report for this and then download the results using the [Report Executions](https://falconpy.io/Service-Collections/Report-Executions.html) service collection.

Thank you for the question!
